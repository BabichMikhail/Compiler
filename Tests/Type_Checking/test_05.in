var a: integer = 10;
	b : double = 10.0000;
begin  
	while a = b do begin
	end;
end.
