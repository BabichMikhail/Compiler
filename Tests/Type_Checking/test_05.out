begin
   while
         b
      =
         a
   do
      begin
      end
end
